web/js/blocks_rts.js

```diff
@@ -0,0 +1 @@
+../../funblocks-base/src/Main.jsexe/rts.js
```

Comment:
Is this any different from the existing rts.js?  I think the RTS is always the same.